src/libs/ajax/ajax-common.ts

```diff
@@ -243,3 +243,6 @@ export const fetchWDS = (wdsProxyUrlRoot) =>
 
 export const fetchCbas = (cbasProxyUrlRoot) =>
   _.flow(withUrlPrefix(`${cbasProxyUrlRoot}/`), withRetryAfterReloadingExpiredAuthToken)(fetchOk);
+
+export const fetchCromwellApp = (cromwellAppProxyUrlRoot) =>
+  _.flow(withUrlPrefix(`${cromwellAppProxyUrlRoot}/`), withRetryAfterReloadingExpiredAuthToken)(fetchOk);
```

Comment:
I noticed that these 3 functions (fetchWDS, fetchCbas, fetchCromwellApp) are identical with the below structure:
```javascript
export const fetchFromProxy = (proxyUrlRoot) => 
  _.flow(withUrlPrefix(`${proxyUrlRoot}/`), withRetryAfterReloadingExpiredAuthToken)(fetchOk);
```

Is there a reason for us to duplicate these? Also, the trailing slash seems a bit minor to incorporate... Wondering if it would be better to adopt one of the below instead of the above:
```javascript
export const fetchFromProxy1 = withRetryAfterReloadingExpiredAuthToken(fetchOk);
export const fetchFromProxy2 = 
  _.flow(fn => withUrlPrefix(_, fn), withRetryAfterReloadingExpiredAuthToken)(fetchOk);
```
which could be used as follows:
```javascript
fetchFromProxy1(`${cbasProxyUrl}/api/batch/v1/...`, requestOptions)
fetchFromProxy2(cbasProxyUrl)('/api/batch/v1/...', requestOptions)
```

Reply:
Yeah I don't see any downside of combining them into a more DRY general `fetchFromProxy` method